src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala

```diff
@@ -29,10 +29,10 @@ case class KhermesConsoleHelper(client: KhermesClientActor) {
   lazy val reader = createDefaultReader()
 
   parseLines(
-    firstLoad(AppConstants.KhermesConfigNodePath),
-    firstLoad(AppConstants.KafkaConfigNodePath),
-    firstLoad(AppConstants.TemplateNodePath),
-    firstLoad(AppConstants.AvroConfigNodePath)
+    firstLoad(AppConstants.GeneratorConfigPath),
```

Comment:
this method "firstLoad" was created by @eruizgar if my memory don't fail. I think that it creates paths if there are not created the first time. Check with him for more information.